### ISSUE TITLE
Replicate Table not providing all data

### ISSUE THREAD

**Issue by ehermosillo** [USGS-R/WQ-Review#139]
Hello.

I have noticed that when I pull data for a Replicate Table in WQ-Review, not all values are provided, however they are in the database as I can see them if I create an excel spreadsheet from QWDATA. 

Thank you,

**Comment by jlwilson-usgs** [comment]
Yeah, I have the same problem. I have 13 replicate samples that I can see if I do a wide table, but when I try to populate the sites, no replicate tables, boxplots, or timeseries plots are displayed. Just a blank screen.

**Comment by tjmills** [comment]
@ehermosillo I apologize for not responding sooner.  I was on paternity leave when this issue was posted and it slipped my mind after I returned. If you are still having problems let me know

**Comment by tjmills** [comment]
@jlwilson 

So a common problem is that the environmental sample of the Env-Rep pair is not coded correctly. Check that your environmental sample associated with your replicate is coded as Medium Code  = WS(or WG, etc) and Sample Type Code = 7 or 5

A number of centers/projects will leave the environmental sample coded as 9, which is not correct. The Env-Rep pairing acts on that sample type code

**Comment by tjmills** [comment]
@ehermosillo @jlwilson 

Feel free to send me your saved data pull (use the save option in the GUI) and I can take a look

**Comment by jlwilson-usgs** [comment]
Sample type is 7 but the medium code is WGQ since its a QA/QC sample.

**Comment by tjmills** [comment]
OK, so the "proper" coding for an environmental-replicate sample pair in NWIS for a groundwater sample is:

Environmental Sample:
MEDIUM_CD = WG
SAMPLE_TYPE_CD = 7

Replicate sample:
MEDIUM_CD = WGQ
SAMPLE_TYPE_CD = 7

Is that how you have your data coded?

This is a nice tip sheet for coding:
http://nwis.usgs.gov/nwisdocs5_2/qw/QW-TipSheet5.11.pdf

**Comment by tjmills** [comment]
Also, you are not the first to have this issue, it seems the coding conventions vary a fair bit across the GS and the "proper" method was not communicated well

**Comment by jlwilson-usgs** [comment]
Nope, looks like I screwed that up :)

Thanks, Joe.

**Comment by tjmills** [comment]
FYI, there is a new 2.0 version that is in testing as of last week if you want to check it out.

[WQReview 2.0](https://github.com/tmills-usgs/WQ-Review_refactor)

Pretty major overhaul that I think will make some folks happy. If your interested let me know and I will send you a link to a short video presentation

**Comment by ehermosillo** [comment]
@tmills-usgs Please let me know if you received my email responding to this thread. Thank you!
